spec/controllers/posts_controller_spec.rb

```diff
@@ -1037,7 +1037,7 @@
 
         login_as(user)
         expect(post).to be_taggable_by(user)
-        expect(post).to receive(:build_new_reply_for).with(user).and_call_original
+        expect(post).to receive(:build_new_reply_for).with(user, {}).and_call_original
```

Comment:
I don't _think_ you need these, since you gave it a default value?

Reply:
I do actually! I don't for generic calls to post.build_new_reply_for() because they can just use the default value, but this is checking what arguments the controller calls the method with, and the controller does in fact call it with both arguments.

Reply:
Aha, okay